Common/Symbol.cs

```diff
@@ -174,6 +174,15 @@ public bool IsCanonical()
                 ID.Date == SecurityIdentifier.DefaultDate;
         }
 
+        /// <summary>
+        /// Method returns true, if symbol is a universe symbol
+        /// </summary>
+        /// <returns>true, if symbol is a universe symbol</returns>
+        public bool IsUniverseSymbol()
+        {
+            return Value.Contains("-UNIVERSE");
```

Comment:
This can't be guaranteed to be correct in either case (universe or non-universe).